src/database/upgrade/24/openfire_hsqldb.sql

```diff
@@ -1,3 +1,3 @@
-ALTER TABLE ofMucConversationLog ADD COLUMN stanza LONGVARCHAR NULL;
+-- ALTER TABLE ofMucConversationLog ADD COLUMN stanza LONGVARCHAR NULL;
```

Comment:
It could/should probably just be deleted.  The table is created with the column with the correct type and size.  The original update index for MIX/MAM was 24, but Dave moved it to 27, I'm guessing as part of a wider openfire merge.